Classes/Controller/AbstractController.php

```diff
@@ -162,4 +165,35 @@ protected function resolveView()
 	{
 		return null;
 	}
+
+
+	/**
+	 * Injects the Configuration Manager and is initializing the framework settings
+	 *
+	 * @param \TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface $configurationManager Instance of the Configuration Manager
+	 */
+	public function injectConfigurationManager( ConfigurationManagerInterface $configurationManager )
+	{
+		// set ConfigurationManager and get settings like the FrontendConfigurationManager
+		$this->configurationManager = $configurationManager;
```

Comment:
If you need the code from the original method nevertheless, use `parent::injectConfigurationMananger( $configurationManager);` instead